README.md

```diff
@@ -0,0 +1,87 @@
+![Aptabase](https://aptabase.com/og.png)
+
+# Unreal Engine SDK for Aptabase
+
+Instrument your Unreal Engine project with Aptabase, an Open Source, Privacy-First and Simple Analytics for Mobile, Desktop and Web Apps.
+
+## Install
+
+Install the SDK via the [Unreal Marketplace](TODO) or clone this repository inside your `Plugins` folder and compile it (requires C++ based project).
+
+## Enabling
+
+The first thing you will need to do is to enable the plugin in your project.
+
+1. Navigate to `Toolbar menu → Edit → Plugins` and open the `Plugins` tab.
```

Comment:
1,2,3,4

Reply:
![image](https://github.com/aptabase/aptabase-unreal/assets/21221169/c4c46957-ccdf-47fe-9f77-e9ff019921df)

Markdown automatically transforms them into the correct index when the page, but if you think people will read the raw .md files, I can make the adjustements

Reply:
Learned something new today :)